src/IO/Logger.cpp

```diff
@@ -282,8 +282,33 @@ void Logger::writeData(std::ofstream &fileStream, const sensorsData &currentStat
 	fileStream << currentState.sbg.deltaAngleX << sep;
 	fileStream << currentState.sbg.deltaAngleY << sep;
 	fileStream << currentState.sbg.deltaAngleZ << sep;
+#endif
+
+	// Initialization data
+#if USE_LOGGER
+    fileStream << currentState.loggerIsInitialized << sep;
+#endif
 
+#if USE_SOCKET_CLIENT
+    fileStream << currentState.client.isInitialized << sep();
```

Comment:
Is there a reason for `sep()` being different for this one?

Reply:
Oops, my bad